src/main/target/common_pre.h

```diff
@@ -242,4 +242,8 @@
 #define USE_OSD_STICK_OVERLAY
 #define USE_ESCSERIAL_SIMONK
 #define USE_SERIAL_4WAY_SK_BOOTLOADER
+#ifndef STM32F7
```

Comment:
Since a big part of this is dependent on the MCU architecture, I think the `#if defined(STM32F4)` block earlier in this file is a better place for this to go.

(Also, does this work on F3? We _do_ have F3 with > 256kB flash, we just declare them as such at the moment.)

Reply:
The code does run on F3 but I haven't done testing with actual ESCs (only have a spare SPRACNINGF3 board lying around). Timing could be challenging. Should I then have an ``#if (FLASH_SIZE > 256)`` in the F3 specific section to allow people to try it out? Or would it then be simpler to leave as is?

Reply:
I think it's best to add it to `#if defined(STM32F4)` for now, and `#if defined(STM32F7)` once it's been ported to run on F7. The only F3 that has > 256 kB flash is the STM32F3DISCOVERY, which isn't used often in actual builds.

Reply:
Ok. Thanks. Done.